src/js/components/Form/index.d.ts

```diff
@@ -1,16 +1,17 @@
 import * as React from "react";
 
-export interface FormProps {
+export interface FormProps<T> {
   errors?: {};
   infos?: {};
   messages?: {invalid?: string,required?: string};
-  onChange?: ((event: React.ChangeEvent) => void);
+  onChange?: (nextState: T) => void;
```

Comment:
I'm wondering if this should be `value` instead of `nextState`.

Reply:
Agreed, I'll update.